language-support/scala/codegen-sample-app/src/main/daml/MyMain.daml

```diff
@@ -590,3 +590,24 @@ template JustMapUser
        aMap: JustMap Int Int
     where
       signatory party
+
+interface InterfaceToMix where
+  getOwner: Party
+  choice InheritedOnly: () with
```

Comment:
only thing I would have thought of would also be good to test is a choice with non-empty argument. However this is sufficient for now

Reply:
Choices are generally tested elsewhere; we will certainly expand the tests based on the overload semantics we eventually choose with the full implementation.